concert-config.yml

```diff
@@ -2011,15 +2011,15 @@ scrapers:
       - name: title
         type: text
         location:
-          selector: div.row_events > a.event.w-inline-block > div.event-lead > div.event-title-alt > h3.event-title-text
```

Comment:
@MarkJaroski why did you update this line?

Reply:
Well, it looks like an editor accident. That's very sloppy work on my part. I should have caught that.

Reply:
No worries :) i was just a little confused :D